crates/proof-of-sql/src/proof_primitive/dory/rand_util.rs

```diff
@@ -16,6 +16,7 @@ pub fn test_seed_rng(seed: [u8; 32]) -> impl Rng {
     StdRng::from_seed(seed)
 }
 
+#[allow(dead_code)]
```

Comment:
```suggestion
#[cfg(test)]
```
then, hide any newly unused imports behind `#[cfg(test)]` as well.